src/styles/clock.scss

```
@@ -0,0 +1,43 @@
+body {
+  background-color: #fbf4ff;
+}
+
+.clock {
+  @include position-center;
+  @include circle($clockface-size);
+
+  position: relative;
+  transform: translate(-50%, 25%);
+
+  background: #dbccfc url(../images/Ios_clock.svg) no-repeat center;
```

don't forget to create variables for all colors

```suggestion
  background: $c-color url(../images/Ios_clock.svg) no-repeat center;
```